### PR TITLE
Add wanda demo to pr env

### PR DESCRIPTION
**Pull request by nelsonkopliku**
# Description

With this little change we have also the possibility to interact with checks execution.

https://1707.prenv.trento.suse.com/

---

Besides this also a specific rabbit user for the PR has been specified, otherwise different PRs interfere with each other with the default user being deleted.

### CODE REVIEW

**Comment by abravosuse** [comment]
What should I be looking at here @nelsonkopliku ?

**Comment by nelsonkopliku** [comment]
> What should I be looking at here @nelsonkopliku ?

That the checks execution at https://1707.prenv.trento.suse.com/ works. Previously it always timed out.

**Review by abravosuse — APPROVED** [review]
Code apart, it looks good to me.

File: .github/workflows/pr_env.yaml [hunk 1]
```diff
@@ -110,13 +110,15 @@ jobs:
...
             web_container_image='ghcr.io/trento-project/trento-web:${{ env.PR_NUMBER}}-env' \
+            wanda_container_image='ghcr.io/trento-project/trento-wanda:demo' \
             wanda_container_name='${{ env.PR_NUMBER}}wanda' \
             docker_network_name='${{ env.PR_NUMBER}}net' \
             nginx_vhost_filename='${{ env.PR_NUMBER}}' \
             trento_server_name='${{ env.PR_BASE_URL}}' \
             web_postgres_password='trento' \
             wanda_postgres_password='wanda' \
             rabbitmq_vhost='${{ env.PR_NUMBER}}' \
+            rabbitmq_username='rabbitusr_${{ env.PR_NUMBER}}' \
```
Comment (arbulu89) [inline .github/workflows/pr_env.yaml:121]: Could we maybe simply use `env.PR_NUMBER` or at least revert the order `${{ env.PR_NUMBER}}rabbitusr` as all the variables follow this pattern?

**Review by arbulu89 — APPROVED** [review]
Nice @nelsonkopliku 

PD: Not your fault, and not need to add any change (as it was already this way), but not having any space before the ending `}}` make my eyes bleed XD

**Comment by nelsonkopliku** [comment]
Thanks @arbulu89 for the review.
I don't wanto to see your eyes bleeding :joy: , so here's the commit fixing the interpolation a40ba4c2884c15ed4eaa6b83c2421f10e9ad9bc0

**Review by arbulu89 — APPROVED** [review]
:heart:

**Pull request merged**